Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -647,546 +552,351 @@ protected virtual void Dispose(bool disposing)
             }
         }
 
-        /// <summary>
-        /// Tries to identify the string and return an object of that class.
-        /// </summary>
-        /// <param name="addr">String to parse.</param>
-        /// <param name="result">IPObject to return.</param>
-        /// <returns><c>true</c> if the value parsed successfully, <c>false</c> otherwise.</returns>
-        private static bool TryParse(string addr, out IPObject result)
+        /// <inheritdoc/>
+        public bool TryParseInterface(string intf, out List<IPData> result)
         {
-            if (!string.IsNullOrEmpty(addr))
+            result = new List<IPData>();
```

Comment:
Does it need to be a list? If you return an array, you can use `Array.Empty`

Reply:
We do dynamically add items later on in the code so we can't use an array directly.